lua/caramba/init.lua

```
@@ -59,160 +59,21 @@ M.commands = require('caramba.commands')
 M.tdd = require('caramba.tdd')
 M.consistency = require('caramba.consistency')
 
--- Initialize the module
+-- Main setup function
 function M.setup(opts)
-  -- Merge user options with defaults
-  M.config.setup(opts)
+  -- Load configuration
+  require('caramba.config').setup(opts)
   
```

Luacheck (1.2.0)

[warning] line contains only whitespace

(W611)